go.mod

```diff
@@ -1,13 +1,14 @@
 module github.com/ONSdigital/dp-healthcheck
 
-go 1.12
+go 1.13
```

Comment:
Can you change this back to 1.12 please? Not everyone is on 1.13 yet so this will break when people pull the latest.